backend/blender_script/main.py

```diff
@@ -58,21 +58,15 @@ def insetLens():
         lenscutter.select_set(False)
 
 # initialize empty scene and spawn a lens in it with given parameters
-def startup(SPHR, SPHL, frame, PD):
+def startup(SPHR, SPHL, CYLR, CYLL, AXISR, AXISL, IOR, frame, PD):
```

Comment:
I think ideally what we wanna do is pass a Prescription object to startup, and then inside the function we parse the class for all the params, instead of having a lot of params for this one function.